### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by mstormi**
remove dashboard
remove FIND2
do not delay loading rules

Fixes: #1316 

Signed-off-by: Markus Storm <markus.storm@gmx.net>

### CODE REVIEW

**Review by Confectrician — COMMENTED** [review]
Cant say much about the bash contents alltough they look goos form my rookie perspective.

One small comment about the docs part.

File: docs/openhabian.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ openHABian is designed as a headless system, you will not need a display or a ke
 -   Enjoy!
 
 -   The device will be accessible by its IP or via the local DNS name `openhabiandevice` (or whatever you changed 'hostname' in `openhabian.conf` to)
--   [Connect to the openHAB 2 dashboard](https://www.openhab.org/docs/configuration/packages.html): [http://openhabiandevice:8080](http://openhabiandevice:8080)
+-   [Connect to the openHAB UI](https://www.openhab.org/docs/configuration/packages.html): [http://openhabiandevice:8080](http://openhabiandevice:8080)
```
[the remark this answers is not in the document]
Comment (mstormi) [reply docs/openhabian.md]: omitted it now

**Comment by eikowagenknecht** [comment]
Do I see it correctly that you removed the functionality to add the tiles alltogether? This might not be needed since tiles are still supported. They just have to be configured in a different file now, see https://community.openhab.org/t/oh3-where-is-frontail-and-paper-ui/110792/13 for details. There is also a GitHub issue for that: https://github.com/openhab/openhab-core/issues/1849

**Comment by mstormi** [comment]
see https://github.com/openhab/openhabian/pull/1320#issuecomment-750019244
not sure if the claim that tiles are still supported is correct but for the time being I'll not remove them.
FIND and delayed rules fixes were applied separately.

**Pull request closed without merging**